projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -59,7 +58,9 @@
    @Value("${datajobs.deployment.dataJobBaseImage:python:3.9-slim}")
    private String deploymentDataJobBaseImage;
    @Value("${datajobs.deployment.builder.extraArgs:}")
-   private String extraArgs;
+   private String builderJobExtraArgs;
+   @Value("${datajobs.deployment.builder.imagePullPolicy:IfNotPresent}")
```

Comment:
Shouldn't the default value be in application.properties?

Reply:
I am not sure. I have always put them here I think. But perhaps it is better for the default to be only in application.properties (alternatively in both places or only here) ?
 What is the recommended way? What are the pros/cons ?

Reply:
If someone uses it elsewhere he/she won't be able to benefit from the default value. Also it is easier to see the default values by looking application.properties.

Reply:
> If someone uses it elsewhere he/she won't be able to benefit from the default value

What do you mean by this?

Reply:
If I inject (`@Value("${datajobs.deployment.builder.imagePullPolicy}")`) this property in another class it wouldn't have a default value.

Reply:
Got it. That make sense.